Install/Chroot

```diff
@@ -223,12 +234,12 @@ Chroot images come with only one locale -- **en_US**, and use UTF-8 version of i
 If  you need to change it, type:
 
 {{{
-# cast -r locale
+# cast -r local

```

Comment:
cast -r locale

Reply:
Uncertain how this happened. Not sure of this was a transcription error or an oversight that I missed when I went through this on my own.

In any case, This has been addressed in an updated on my local fork. This should be taken care of in a subsequent pull request.